modules/custom/activity_basics/activity_basics.module

```diff
@@ -42,8 +53,21 @@ function activity_basics_entity_presave(EntityInterface $entity): void {
     // Check if entity change state from "unpublished" to "published".
     // @phpstan-ignore-next-line
     $original = $entity->original;
+
     if (!$original->isPublished() && $entity->isPublished()) {
       _activity_basics_entity_action($entity, 'publish_entity_action');
+
+      // Load all of the group content for this entity.
+      // Check if related entity become published then trig activity creation.
+      /** @var \Drupal\group\Entity\GroupContentInterface[] $group_contents */
```

Comment:
@VBaldych there is PHPStan [warning](https://github.com/goalgorilla/open_social/actions/runs/8064696520/job/22029179024?pr=3790) about this line:
```Variable $group_contents in PHPDoc tag @var does not exist.```

Do you really need this `var` there?
I think issue can be fixed simply by removing this line.

Reply:
Done